### DISCUSSION TITLE
Discord ve telegram +1000 puanları nasıl alabiliriz?

### DISCUSSION THREAD

**Discussion by alicanharmanci** [IBAX-io/go-ibax#1339] [Soru Cevap - Türkçe] [answered]
Cevap var mı acaba?

**Comment by Trajedi032** [comment]
Bende cevap bekliyorum ama yeterli yardım yok

**Comment by Omertastan2036** [comment]
Alan yok diye biliyorum

**Comment by Yusufknk** [comment, marked as answer]
Arkadaşlar bunun için yöneticilerle görüşüyoruz. İlk başta form olarak yapacaklardı ama birden fazla bot hesap ihtimali olduğu için bu yöntemi elediler.

**Reply by coinsinov** [reply]
iBAX Twitter hesabını takip 2000 puanımı alamadım daha sonra tekrar deneyin diyor üstad yardımcı olur musunuz? Teşekkür ederim.

**Reply by Yusufknk** [reply]
Farklı bir tarayıcı denedin mi?

**Reply by coinsinov** [reply]
Crome'dan denedim sadece
